components/List.tsx

```diff
@@ -0,0 +1,14 @@
+interface Props {
+  lists: { content: string }[];
+}
+export function List({ lists }: Props) {
+  const className = "grid grid-cols-1 gap-1 list-decimal";
+  return (
+    <ul className={className}>
+      {lists.map((list, idx) => {
```

Comment:
**praise:** We should not use order in the list as a key for the component. It is not recommended. Probably the best way (here) is to hash or encode each item unless we generate id for each item.

Reply:
Now, we're receiving the data without an id. do i use the hashed content?

Reply:
Okay